tox.ini

```diff
@@ -14,11 +14,11 @@ commands=
     flake8 redo
     py.test -v --cov=redo --cov-report term-missing --doctest-modules 
 
-[testenv:py27-coveralls]
-deps=
-    python-coveralls==2.4.3
-commands=
-    coveralls
```

Comment:
sooo... we're losing coverage support?

Reply:
We haven't had coverage support since (I'm guessing) the switch from travis to taskcluster. I'm just removing some dead code.